src/assets/styles/mixins/layout.scss

```diff
@@ -149,6 +149,10 @@ $scrollbar-offset: 20px;
     width: 40px;
   }
 
+  .spacer-root {
```

Comment:
After some time it gets hard to understand why CSS were added to the global space. It's better to add to component file even if you have to use `::ng-deep`.